src/transformStateWithClones.js

```diff
@@ -8,6 +8,38 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  let clone = {};
```

Comment:
You can copy state via spread syntax, this will allow you to reduse amount of code later
```suggestion
  let clone = { ...state };
```